components/VM_Arm/src/main.c

```diff
@@ -927,17 +951,6 @@ static int load_linux(vm_t *vm, const char *kernel_name, const char *dtb_name, c
     return 0;
 }
 
-void parse_camkes_linux_attributes(void)
-{
-    linux_ram_base = strtoul(linux_address_config.linux_ram_base, NULL, 0);
```

Comment:
It we can avoid calling `strtoul()` at runtime, that's seem really nice.

Reply:
Agreed!